### DISCUSSION TITLE
focus_on_activate, mouse position.

### DISCUSSION THREAD

**Discussion by Memoframe** [hyprwm/Hyprland#4476] [Q&A] [answered]
Do we have an option to **not** move mouse to center of the screen but keep it in the same place?

**Comment by Memoframe** [comment, marked as answer]
In **general** section of config:
no_cursor_warps = true

**Comment by Rabcor** [comment]
Memoframe's answwor no longer works, now it's like this:

```
cursor {
  no_warps = true
}
```
